### DISCUSSION TITLE
[Feature] Group Select based on Map Location

### DISCUSSION THREAD

**Discussion by Aseeef** [immich-app/immich#11130] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

**The feature request:**
In the "map" tab of immich, photos are grouped by location. I am requesting adding functionality so that you are able to select all these images together and perform actions like delete, add to album, set location, and basically all the things you can do when you select a group of images regularly like shown below:
![image](https://github.com/user-attachments/assets/77ba7c86-28f3-49bb-b886-8d3bc36d8cae)

**Motivation:**
For whatever reason, (no fault of immich), some of my photos had corrupted GPS meta such that it has the wrong location.
![image](https://github.com/user-attachments/assets/7ab67f41-9a1d-452d-9ea8-83b515cf2408)
I would like to be able to select the entire group of photos in this location and move them to the correct place.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Dupe of #4519/#9159
